CHANGELOG.md

```diff
@@ -2,110 +2,144 @@
 
 ## 4.0.0 - UNRELEASED
 
-- BREAKING: Map `multi-merge` utility and all `add-*` functions
-  for adding one or more token maps (eg `add-sizes`, `add-colors`, etc)
-  now perform a deep merge of the token maps.
-  Previously a shallow merge was performed,
-  [which could result in removing nested values from maps](https://github.com/oddbird/accoutrement/issues/50).
+This is a major update, moving over to Sass modules,
+and removing some features that no longer seem necessary.
 
-- [Tokens][token]:
+In addition to fixing a bug with the handling of
+non-alias `#` symbols, and implementing deep map merges
+where appropriate, we also added some functionality around
+compiling maps for export, and working with different
+color palettes.
 
-  - NEW: `map-compile` and `map-compile-with` functions for compiling
-    tokens in Accoutrement maps and formatting them for static exports.
+- BREAKING: Requires Sass modules, and other recent features of Dart Sass.
+  Legacy versions of Sass (Node & Ruby) are no longer supported.
+- BREAKING: The old "init" module, which provided lightweight
+  browser normalization, is no longer included. We now recommend
+  using [CSS Remedy](https://github.com/jensimmons/cssremedy). (_Note: If
+  making the switch from Accoutrement/Init to CSS Remedy, you may notice changes
+  in vertical spacing and the displaying of SVGs._)
+- BREAKING: The old "core" module has been broken into individual modules
+  for [utilities][utils] (extending several Sass modules),
+  [tokens][token] (the map syntax parser),
+  [variables][vars] (converting Sass variables & maps to CSS custom properties),
+  and [ratios][ratio] (for typographic scales and layout aspect ratios).
```

Comment:
It seems like maybe we should get rid of these absolute links and replace them with relative links, e.g.:

```suggestion
  for [utilities](utils.html) (extending several Sass modules),
  [tokens](tokens.html) (the map syntax parser),
  [variables](vars.html) (converting Sass variables & maps to CSS custom properties),
  and [ratios](ratio.html) (for typographic scales and layout aspect ratios).
```

Reply:
I started to do this @jgerigmeyer but then I realized that I had forgotten that people could be reading and interacting with this in the on our site  _and_ here in the repo. For the links to work in both places they would all have to be absolute, right?

Reply:
Ah, good point! I knew there was a reason why we'd used absolute links in the past 😂